inductiva/stellarators/__init__.py

```diff
@@ -1,2 +1,2 @@
 #pylint: disable=missing-module-docstring
-from .scenarios import StellaratorCoils
+from .coils import StellaratorCoils
```

Comment:
Check with @hpenedones if this is where we want to have this.
I am not sure.

Reply:
The general idea that I want to pass is that the modules `stellarators`, `fluids`, `molecules` are already specific to scenarios. So stating scenarios is redudant.

Reply:
It can be in a separate PR if you prefer, but the package for stellarators should be called plasmas. For consistency with WindTunnel (which is a simulated machine, like Stellarators) is under fluids, not under a package called "windtunnel".

Reply:
I am gonna do it next :)